akka-http/src/main/scala/akka/http/scaladsl/server/directives/RouteDirectives.scala

```diff
@@ -49,6 +49,14 @@ trait RouteDirectives {
   def complete(m: => ToResponseMarshallable): StandardRoute =
     StandardRoute(_.complete(m))
 
+  /**
+   * Completes the request using the given arguments.
+   *
+   * @group route
+   */
+  def complete[T](status: StatusCode, m: ToResponseMarshallable): StandardRoute =
```

Comment:
This should probably be
```suggestion
  def complete[T](status: StatusCode, v: T)(implicit marshaller: ToEntityMarshaller[T]): StandardRoute =
```
and then we should put those together so we can call the regular `ctx.complete(ToResponseMarshallable)`